deploy/infra/groups/orchestration.bicep

```diff
@@ -27,6 +27,8 @@ param uamiName string = ''
 param pipelineResourceGroupName string
```

Comment:
If all the resources and modules in this bicep file need to be skipped based on the condition of pre-existing batch account, then can we add the "if(deployBatchAccount) ..." condition one level up to the main.bicep instead of adding the condition multiple times for each resource/module within the orchestration.bicep?

Reply:
yeah, i wanted to do that, but there are couple of resource which we are not skipping.